src/client/app/components/unit/CreateUnitModalComponent.tsx

```diff
@@ -33,6 +33,8 @@ export default function CreateUnitModalComponent() {
 		preferredDisplay: true,
 		secInRate: 3600,
 		suffix: '',
+		defaultMeterMinimumValue:Number.MIN_SAFE_INTEGER,
+		defaultMeterMaximumValue:Number.MAX_SAFE_INTEGER,
```

Comment:
Please use 
import { MAX_VAL,  MIN_VAL } from '../../redux/selectors/adminSelectors';
Both refer to the same value but the above import is used in places where ever the requirement is for max and min values. So please use the above import to mantain uniformity in the code